packages/shineout/src/form/__example__/test-001-base-form.tsx

```
@@ -0,0 +1,170 @@
+
+const ComponentB2 = () => {
+  useEffect(() => {
+    console.log('B2 mounted');
+  }, []);
+
+  return (
+    <Form.Item label='B2'>
+      <Form.FieldSet name='a.spec_info_list' defaultValue={[{ 'b-1': 3, 'b-2': 4 }]}>
+        {({}) => {
+          return (
+            <div>
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)